src/index.ts

```diff
@@ -96,6 +96,13 @@ export function definedMap<T, U>(x: T | undefined, f: (xx: T) => U): U | undefin
     return f(x);
 }
 
+/**
+ * If `x` belongs to the enum `e`, return `true`.  Otherwise, return `false`.
+ */
+export function isEnumValue<T>(e: T, x: unknown): x is T[keyof T] {
+    return (Object.keys(e) as Array<keyof T>).map(k => e[k]).some(v => v === x as T[keyof T]) ?? false;
```

Comment:
Why is the `?? false` necessary?  The `===` always produces boolean.

Reply:
I didn't see the close paren - forget the part about `===`.  But `.some` also produces a boolean, so the `??` should still not be necessary.

Reply:
Quick drive-by comment, isn't
```ts
(Object.keys(e) as Array<keyof T>).map(k => e[k])
```
just
```ts
Object.values(e)
```
?

Reply:
@ivoelbert yes it is, but this project's target ES version doesn't support `Object.values(e)` so I needed to use `keys` and map the dereferencing instead.  Also why I couldn't use `[].includes(x)` which would've simplified even more.